collector_test.go

```diff
@@ -207,4 +170,64 @@ var _ = Describe("RedigoCollector", func() {
 		close(done)
 	})
 
+	It("should test time of method", func() {
+		var service RedigoService
+		var metric dto.Metric
+
+		Expect(service.ApplyConfiguration(Configuration{
+			Address: "localhost:6379",
+		})).To(BeNil())
+		Expect(service.Start()).To(BeNil())
+
+		defer service.Stop()
+
+		ctx, cancel := context.WithCancel(context.Background())
+		onSubscribed := func() error {
+			Expect(service.Publish(ctx, "test-06", []byte("test"))).To(Succeed())
+			return nil
+		}
+
+		service.Subscribe(ctx, onSubscribed, func(channel string, data []byte) error {
+			cancel()
+			return nil
+		}, "test-06")
+
+		Expect(service.Collector.methodDuration.With(prometheus.Labels{
+			"method":  "Do",
+			"command": "PUBLISH",
+		}).Write(&metric)).To(BeNil())
+
+		Expect(metric.GetCounter().GetValue()).To(BeNumerically(">", 0.0))
+	})
+
+	It("should test total calls of command", func() {
+		var service RedigoService
+		var metric dto.Metric
+
+		Expect(service.ApplyConfiguration(Configuration{
```

Comment:
I think this should be in a `BeforeEach` function

Reply:
Please, see it again. :grin: